lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -403,6 +416,18 @@ class FatMemManager : public OpSemMemManager {
   FatMemValTy zeroedMemory() const override {
     return mkFatMem(m_mem.zeroedMemory());
   }
+
+  Expr getFatData(PtrTy p, unsigned SlotIdx) override {
+    assert(strct::isStructVal(p));
+    assert(SlotIdx < 2);
```

Comment:
would be nice to factor out hard coded constant `2` into a constant just like `slotBitWidth`

Reply:
done